File: C++code/MString.cpp
#include "MString.h"

void MString::setValue(const string& newValue) { 
    value = newValue; 
}

string MString::getValue() const {  
    return value;
}

int MString::length() const {
    int count = 0;
    for (char symbol : value) {  
        count++;
    }
    return count;
}

void MString::removeFive() {
    string newValue = "";
    for (char symbol : value) {  
        if (symbol != '5') {
            newValue += symbol;
        }
    }
    value = newValue;           
}

File: C#code/MString.cs
using System;

namespace MyProgram
{
    public class MString
    {
        private string value = ""; 

        public void SetValue(string newValue) 
        {
            value = newValue;
        }

        public string GetValue()
        {
            return value;
        }

        public int Length()
        {
            int count = 0;
            foreach (char symbol in value)
            {
                count++;
            }
            return count;
        }

        public void RemoveFive()
        {
            string newValue = "";
            foreach (char symbol in value)
            {
                if (symbol != '5')
                {
                    newValue += symbol;
                }
            }
            value = newValue;
        }
    }
}